### DISCUSSION TITLE
Geo Dome

### DISCUSSION THREAD

**Discussion by bostonkassidy** [EnviralDesign/GeoPix#43] [Show and tell]
https://user-images.githubusercontent.com/47906726/128558145-3b66e7f2-247e-4228-b650-461862814ed7.MOV

https://user-images.githubusercontent.com/47906726/128558165-73e7127b-cb70-457a-945e-71d389c005e6.MOV

Just wanted to share some early pics of a project were working on. Its a 35' geodesic dome, fully pixel mapped on the interior using geopix. Super excited about how its coming along. Ill post more pics as we start to build in the next few weeks.

**Comment by EnviralDesign** [comment]
Wow thanks for sharing! I am just now catching this notification. This is looking great :) Please do share any updates or finals if you are able to.
